{"uuid": "1cdc6fc3-1a1e-4a1c-99c8-6945790d186f", "children": ["52f37a9a-e7d5-420e-a783-806115196452", "ea407375-1785-4234-b559-0bc7a6f58946", "1e96517f-abac-45b8-a97b-684dc7de39f9"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource\n\n", "trace": "  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/.venv/lib/python3.9/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/.venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1122, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/.venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 917, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/conftest.py\", line 36, in driver\n    request.cls.driver = webdriver.Remote(url, capabilities)\n  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/.venv/lib/python3.9/site-packages/appium/webdriver/webdriver.py\", line 274, in __init__\n    super().__init__(\n  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 266, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/.venv/lib/python3.9/site-packages/appium/webdriver/webdriver.py\", line 369, in start_session\n    response = self.execute(RemoteCommand.NEW_SESSION, parameters)\n  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 418, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/rucindrea/workspace/courses/AltTicTacToe/AltTicTacToeInstrumented/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 243, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1646329655779, "stop": 1646329655799}], "start": 1646329655779, "stop": 1646329655937}